draft-ietf-babel-rfc6126bis/draft-ietf-babel-rfc6126bis.xml

```diff
@@ -744,6 +753,15 @@ a small multiple of the value received in the IHU.</t>
 neighbour's cost (<xref target="cost-computation"/>) and runs the route
 selection procedure (<xref target="route-selection"/>).</t>
 
+<t>The bidirectional reachability detection is a prerequisite for the
+routing information exchange and processing. Specifically, a node MUST ignore:
```

Comment:
I'm not sure this is the best solution. An implementation could still learn useful information from updates from a host that it has not heard Hellos for. Those routes must have an infinite metric but once we do get the hello we could immediately start using them instead of waiting for another update or asking for one explicitly.